bigquery_etl/glean_usage/events_unnested.py

```diff
@@ -13,12 +13,9 @@ class EventsUnnestedTable(GleanTable):
 
     def __init__(self):
         """Initialize events_unnested table."""
+        GleanTable.__init__(self)
         self.target_table_id = TARGET_TABLE_ID
         self.prefix = PREFIX
-        self.custom_render_kwargs = {}
-        self.no_init = False
+        self.no_init = True
+        self.per_app_id_enabled = False
         self.cross_channel_template = "cross_channel_events_unnested.view.sql"
-
-    def generate_per_app_id(self, project_id, baseline_table, output_dir=None):
-        """Generate the baseline table query per app_name."""
-        logging.info("generate_per_app_id() not implemented for EventsUnnestedTable")
```

Comment:
This is now handled by a `per_app_id_enabled` check in the body of the upstream function.